SourceCode/Power/ftt_p_main.py

```diff
@@ -810,6 +808,16 @@ def solve(data, time_lag, iter_lag, titles, histend, year, domain):
             data['MWIY'][:, :, 0] = (data_dt['MWIY'][:, :, 0]
                                      + (data['MEWI'][:, :, 0] * dt * data['BCET'][:, :, c2ti['3 Investment ($/kW)']] / 1.33))
             
+            sector_coupling_assumps = get_sector_coupling_dict(data, titles)
+            
+            battery_lifetime = 12 #assuming 12 years
+            
+            capacity_batteries_current_year = data["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            capacity_batteries_last_year = time_lag["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            
+            single_year_capacity = capacity_batteries_current_year - capacity_batteries_last_year
```

Comment:
Minor comment: try to be as specific as possible in your variable names: this is additions of capacity (so varname could be yearly_cap_additions.